src/chatdMsg.h

```diff
@@ -513,7 +513,8 @@ class Message: public Buffer
     {
         kInvalid              = 0xff,
         kRichLink             = 0x00,
-        kGeoLocation          = 0x01
+        kGeoLocation          = 0x01,
```

Comment:
Do you know the purpose of the `0x02`?? it would be good to add it here, just for documentation. Maybe Lyubo can tell you.

Reply:
guess 0x02 is reserved for something or might just be an oversight. Will check.